engine/jni/boot.cc

```diff
@@ -219,6 +219,7 @@ bool Bootstrap::LoadAnalysisModule()
         idx_end = g_module_path + strlen(g_module_path) - 1;
     char keyword[kBlahSizeTiny];
     strncpy(keyword, idx_bgn, idx_end - idx_bgn + 1);
+    keyword[idx_end - idx_bgn + 1] = 0x00;
```

Comment:
Thanks for the bug report!

Could you help to change the statement to:
```
keyword[idx_end - idx_bgn + 1] = 0;
```

That's just a minor modification for style consistency.

Then the patch can be directly merged.

Thanks!